confluent_kafka/src/confluent_kafka.c

```diff
@@ -1131,6 +1131,37 @@ static void error_cb (rd_kafka_t *rk, int err, const char *reason, void *opaque)
 	CallState_resume(cs);
 }
 
+static void throttle_cb (rd_kafka_t *rk, const char *broker_name, int32_t broker_id,
+        int throttle_time_ms, void *opaque) {
+
+	Handle *h = opaque;
+	PyObject *eo, *result;
+	CallState *cs;
+
+	cs = CallState_get(h);
+	if (!h->throttle_cb) {
+		/* No callback defined */
+		goto done;
+	}
+
+	eo = Py_BuildValue("{s:s,s:i,s:i}", "broker_name", broker_name, "broker_id", broker_id,
```

Comment:
I think we'll want to a use properly classed object here instead of a dict.
You can define the class in __init__.py and call its constructor with this dict.